src/_kw

```diff
@@ -456,11 +456,19 @@ _kw_self-update()
 _kw_s() { _kw_ssh }
 _kw_ssh()
 {
+  local to=''
+
+  if ((words[(I)--send|--get])); then
+    to='--to[specify the destination path]: : '
+  fi
```

Comment:
@davidbtadokoro could you review this part?

Reply:
Hi @rodrigosiqueira and @aquilamacedo.
The implementation looks good to me!

We can further refine it by allowing completions for files/dir after `--send` and `--to` (when using `--get`). For example, hitting tab with `kw ssh --send` or `kw --get <path> --to` in the prompt would suggest completions for the host filesystem. If you want to follow this suggestion, you can apply this diff on top of this commit:
```
diff --git a/src/_kw b/src/_kw
index 7f478da..9dbe398 100644
--- a/src/_kw
+++ b/src/_kw
@@ -462,15 +462,17 @@ _kw_ssh()
 {
   local to=''
 
-  if ((words[(I)--send|--get])); then
+  if ((words[(I)--send])); then
     to='--to[specify the destination path]: : '
+  elif ((words[(I)--get])); then
+    to='--to[specify the destination path]: :_files'
   fi
 
   _arguments : \
     '(--verbose)--verbose[enable verbose mode]' \
     '(-s --script -c --command --send --get)'{-s,--script}'[execute a bash command in the target machine]: : ' \
     '(-c --command -s --script --send --get)'{-c,--command}'[execute a bash script in the target machine]: : ' \
-    '(-s --script -c --command --get)--send[send a file or directory to the remote machine]: : ' \
+    '(-s --script -c --command --get)--send[send a file or directory to the remote machine]: :_files' \
     '(-s --script -c --command --send)--get[get a file or directory from the remote machine]: : ' \
     $to
 
```

I think Zsh also supports completions remote filesystems (try `scp`, for example), and I think this is possibly feasible based on [this](https://github.com/zsh-users/zsh/blob/master/Completion/Unix/Type/_remote_files), but it seems that background ssh connections are opened and closed for each <TAB> hit and maybe this is too much of a hassle 😅.

Reply:
Let's open an issue to handle this. @aquilamacedo , please check David's comment and open an issue (add as many details as possible).